src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -1061,7 +1061,7 @@ void DirectoryService::RunConsensusOnFinalBlock()
 
         if (isVacuousEpoch)
         {
-            LOG_MESSAGE(
+            LOG_GENERAL(
                 "I am going to sleep for 5 seconds for vacuous epoch.");
             this_thread::sleep_for(chrono::seconds(5));
```

Comment:
Do not use magic number

Reply:
This part is commented out, I used global replace for changing LOG_MESSAGE to LOG_GENERAL.

Reply:
Yes understood, but is this part of code is actually discarded by someone by commenting, rather than me. You can find who wrote and discarded this line of code, I'm not sure whether I should be the person delete it

Reply:
I see. I saw the /* */

Reply:
Ok for this. Created issue #268